--- a/PKGBUILD	2021-08-09 14:03:48.107472183 -0300
+++ b/PKGBUILD	2021-08-09 14:46:54.959563141 -0300
@@ -37,11 +37,12 @@
     git checkout ${pkgver}-stable
 
     #Build temporary binaries to generate needed files for mono support
-    scons platform=x11 tools=yes module_mono_enabled=yes mono_glue=no
+    scons -j $(nproc) platform=x11 tools=yes module_mono_enabled=yes mono_glue=no
+    xvfb-run -a -n 97 -s "-screen 0 1600x1200x24" \
     bin/godot.x11.tools.64.mono --generate-mono-glue modules/mono/glue
 
     # Build normal binaries
-    scons platform=x11 target=release_debug tools=yes module_mono_enabled=yes bits=64
+    scons -j $(nproc) platform=x11 target=release_debug tools=yes module_mono_enabled=yes bits=64
 }
 
 package() {
