

diff --git a/PKGBUILD b/PKGBUILD
index c5dbc208bbccab1573c6..01e3c8e01307bbf0ade8 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -5,7 +5,6 @@ pkgver=1.0.1
 pkgrel=1
 pkgdesc="Generic command-line automation tool (no X!)"
 arch=('x86_64' 'aarch64')
-depends=('libevdevplus' 'libuinputplus')
 makedepends=('cmake' 'ninja' 'scdoc')
 url="https://github.com/ReimuNotMoe/ydotool"
 license=('AGPL3')
@@ -24,7 +23,6 @@ build() {
     -DCMAKE_INSTALL_PREFIX=/usr \
     -DCMAKE_INSTALL_MANDIR=/usr/share/man \
     -DCMAKE_BUILD_TYPE=Release \
-    -DCXXOPTS_ENABLE_INSTALL=OFF \
     -G Ninja \
     -S "$pkgname-$pkgver" -B build
   ninja -C build
