diff --git a/PKGBUILD b/PKGBUILD
index 2a21902..f837020 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -2,15 +2,17 @@
 
 pkgname=anbox-git
 _pkgname=anbox
-pkgver=r1292.7f1bfaf
+pkgver=r1348.ad377ff
 pkgrel=1
 epoch=1
 arch=('x86_64')
 url="http://anbox.io/"
 license=('GPL3')
 pkgdesc="Running Android in a container"
-depends=('lxc' 'sdl2_image' 'protobuf' 'anbox-image' 'libsystemd' 'boost-libs')
-makedepends=('cmake' 'git' 'glm' 'lxc' 'sdl2_image' 'protobuf' 'boost' 'properties-cpp' 'gtest' 'python')
+depends=('lxc<=1:4.0.6' 'sdl2_image' 'protobuf' 'anbox-image' 'libsystemd'
+'boost-libs')
+makedepends=('cmake' 'git' 'glm' 'lxc<=1:4.0.6' 'sdl2_image' 'protobuf' 'boost'
+'properties-cpp' 'gtest' 'python')
 source=("git+https://github.com/anbox/anbox.git"
   "git+https://github.com/google/cpu_features.git"
   "git+https://github.com/Kistler-Group/sdbus-cpp.git"
@@ -46,6 +48,7 @@ prepare() {
   # Don't build tests
   truncate -s 0 cmake/FindGMock.cmake
   truncate -s 0 tests/CMakeLists.txt
+  sed -i '1i\#include <cstdint>' "$srcdir/anbox/src/anbox/input/manager.cpp"
 
   git submodule init
   git config submodule.external/cpu_features.url $srcdir/cpu_features

