diff --git a/PKGBUILD b/PKGBUILD
index 363475a..884f76c 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -9,8 +9,15 @@ license=(custom:MIT)
 conflicts=("${pkgname}-git")
 depends=('wayfire>=0.7' cairo glibmm iio-sensor-proxy)
 makedepends=(meson ninja glm git)
-source=("https://github.com/WayfireWM/${pkgname}/releases/download/v${pkgver}/${pkgname}-${pkgver}.tar.xz")
-sha256sums=(01dcb040ced5509cf8f8a1f46dba6f61b613209b76b82550906140266c4921eb)
+source=("https://github.com/WayfireWM/${pkgname}/releases/download/v${pkgver}/${pkgname}-${pkgver}.tar.xz"
+       build_fix.patch::https://gist.githubusercontent.com/dr460nf1r3/e5c52144372e0948229e96e2f1936e35/raw/4d4ab455c9ed781939edc1d07e5fc990f6ebea69/gistfile1.txt)
+sha256sums=('01dcb040ced5509cf8f8a1f46dba6f61b613209b76b82550906140266c4921eb'
+            '7f6cb2ed474b0a74cb96e4706328a869ab777018869f4cb7eba7aebd2b38e3e6')
+
+prepare() {
+  cd $pkgname-$pkgver
+  patch -Np1 -i $srcdir/build_fix.patch
+}

 build () {
    rm -rf build
