diff --git a/PKGBUILD b/PKGBUILD
index 37b2beb..bba291f 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -20,7 +20,7 @@ depends=(lib32-aom lib32-bzip2 lib32-chromaprint lib32-curl lib32-faac lib32-faa
         lib32-nettle lib32-openal lib32-openexr lib32-openjpeg2 lib32-openssl lib32-opus lib32-orc
         lib32-pango lib32-rtmpdump lib32-sbc lib32-soundtouch lib32-spandsp lib32-srt
         lib32-vulkan-icd-loader lib32-wayland lib32-webrtc-audio-processing lib32-wildmidi
-        lib32-x265 lib32-zbar lib32-zvbi gst-plugins-bad)
+        lib32-zbar lib32-zvbi gst-plugins-bad)
 makedepends=(git lib32-bluez-libs lib32-glu lib32-gtk3 lib32-ladspa lib32-libexif lib32-lv2 lib32-vulkan-validation-layers
              meson python lib32-shaderc vulkan-headers wayland-protocols)
 optdepends=('lib32-nvidia-utils: nvcodec plugin')
@@ -62,6 +62,7 @@ build() {
         -D msdk=disabled \
         -D opencv=disabled \
         -D openh264=disabled \
+        -D x265=disabled \
         -D openmpt=disabled \
         -D openni2=disabled \
         -D opensles=disabled \
