diff --git a/PKGBUILD b/PKGBUILD
index e2ba36c..dd0798c 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -8,8 +8,8 @@
 
 _pkgbasename=x264
 pkgname=lib32-x264
-pkgver=0.161.r3039.544c61f
-pkgrel=2
+pkgver=0.163.r3060.5db6aa6
+pkgrel=1
 epoch=3
 pkgdesc='Open Source H264/AVC video encoder (32 bit)'
 arch=('x86_64')
@@ -20,7 +20,7 @@ makedepends=('git' 'nasm' 'lib32-gcc-libs')
 provides=('lib32-libx264' 'libx264.so')
 conflicts=('lib32-libx264' 'lib32-libx264-10bit' 'lib32-libx264-all')
 replaces=('lib32-libx264' 'lib32-libx264-10bit' 'lib32-libx264-all')
-_commit='544c61f082194728d0391fb280a6e138ba320a96'
+_commit='5db6aa6'
 source=("git+https://git.videolan.org/git/x264.git#commit=${_commit}")
 sha256sums=('SKIP')
 
